app/src/main/java/com/example/movieapp/crawler/pojo/Movie.kt

```diff
@@ -0,0 +1,25 @@
+package com.example.movieapp.crawler.pojo
+
+import com.google.gson.annotations.SerializedName
+import java.util.*
+
+/**
+ * Created by Benjamin Vouillon on 08,July,2020
+ */
+data class Movie(
+    val popularity: Float,
```

Comment:
Put the @SerializedName annotation on each field even if the name doesn't change. 
It'll prevent an issue where you would rename the val and break your code. 
Also, put a space after the parenthesis or an linebreak ;-)

Reply:
@SerializedName done
add spaces